src/handlers/local-upload.ts

```diff
@@ -60,6 +61,14 @@ async function pollForAssetReady(filePath: string, asset: Asset) {
   }
 
   if (muxAsset.status === 'ready') {
+
+    let blurDataURL;
+    try {
+      blurDataURL = await createThumbHash(`https://image.mux.com/${playbackId}/thumbnail.png?width=100&height=100`);
```

Comment:
Does it matter that this is going to distort the aspect ratio? Should we just specify height or width to maintain it?

Reply:
no, they're max dimensions. it keeps aspect ratio 
> By default, Mux will preserve the aspect ratio of the video

Reply:
Let's pretend I didn't ask that.